package.json

```diff
@@ -10,7 +10,8 @@
     "build:clean": "rimraf build/",
     "watch": "concurrently \"npm run watch:compile\" \"npm run watch:start\"",
     "watch:compile": "swc src -w -d build -D",
-    "watch:start": "nodemon --watch \"build/**/*\" -e js ./build/index.js"
+    "watch:start": "nodemon --watch \"build/**/*\" -e js ./build/index.js",
+    "prepare": "husky install"
```

Comment:
May be useful add `chmod +x .husky/pre-commit` to this script. Because git will not use the `pre-commit` hook by default if it isn't set as executable.

Reply:
Git will not run the hook if `npm run prepare` is not run before. `husky install` sets the .husky folder to [`core.hooksPath`](https://git-scm.com/docs/githooks#_synopsis), and then Git looks for hooks in that folder and runs it automatically.